pkg/server/server.go

```diff
@@ -29,15 +29,32 @@ import (
 	"os"
 	"path/filepath"
 	"strconv"
+	"strings"
 	"time"
 
+	"github.com/emicklei/go-restful"
+
+	// Enable Profiling
```

Comment:
This doesn't seem to be in the right spot?

Reply:
Thank you openshift-goimports 🤦